interface/billing/sl_eob_invoice.php

```diff
@@ -540,7 +540,7 @@ function updateFields(payField, adjField, balField, coPayField, isFirstProcCode)
                     </div>
                     <div class="form-group col-lg">
                         <label class="col-form-label" for="form_check_date"><?php echo xlt('Check/EOB Date'); ?>:</label>
-                        <input type='text' name='form_check_date' class='form-control datepicker' value=''/>
+                        <input type='text' name='form_check_date' id='form_check_date" class='form-control datepicker' value=''/>
```

Comment:
The OpenEMR fans go crazy even over the small changes!

![2DV](https://user-images.githubusercontent.com/278968/83934503-720a4580-a766-11ea-9f2b-4b08e4ff9534.gif)